webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- Please, kindly remove this eslint disable line of codes, as this has no effect on the codebase.